Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "SciMLSensitivity"
 uuid = "1ed8b502-d754-442c-8d5d-10ac956f44a1"
 authors = ["Christopher Rackauckas <accounts@chrisrackauckas.com>", "Yingbo Ma <mayingbo5@gmail.com>"]
-version = "7.77.0"
+version = "7.76.0"
```

Comment:
?

Reply:
Ah this was picked from the history, updated. Thanks